interop-switch/templates/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "interop-switch.fullname" . }}
+  labels:
+    app: {{ template "interop-switch.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "interop-switch.name" . }}
```

Comment:
We are using fullname for the app label.
Please change too: app: {{ template "interop-switch.fullname" . }}

Reply:
@mdebarros I believe I made all the changes you suggested. Added some additional functionality. Can you please review.